templates/frontend/pages/indexSite.tpl

```diff
@@ -18,6 +18,37 @@
 		</div>
 	{/if}
 
+	{* Announcements *}
+	{if $numAnnouncementsHomepage && $announcements|@count}
```

Comment:
This section appears to come from `templates/frontend/pages/indexJournal.tpl`; would it be worth extracting it into a common template rather than duplicating the code? If a theme previously overrode this template, it would continue to do so and not call on the new template -- so no harm done, I think. (And I suspect it might be best for that new template to go into pkp-lib, since it's probably copypasta between apps.)